grpc/estimator.proto

```diff
@@ -106,14 +120,33 @@ message SettingsResponse {
 // forces
 
 message Cost {
-  optional double total = 1;
-  optional double prior = 2;
-  optional double sensor = 3;
-  optional double force = 4;
-  optional double initial = 5;
+  double total = 1;
```

Comment:
Even though they'll always be returned, I think the custom is to keep everything "optional", not sure why.  Maybe @nimrod-gileadi can comment further.

Reply:
Take away, we don't need to specify `optional` if the field is always set, so the code is fine as is?

Reply:
@erez-tom I removed the `Cost` message and moved its members into `CostResponse`.